Saml.Integration/packages.config

```diff
@@ -14,7 +14,7 @@
   <package id="System.Buffers" version="4.4.0" targetFramework="net461" />
   <package id="System.Net.Http" version="4.3.3" targetFramework="net461" />
   <package id="System.Runtime.CompilerServices.Unsafe" version="4.5.0" targetFramework="net461" />
-  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net461" />
+  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net461" requireReinstallation="true" />
```

Comment:
It might be worth seeing if we can avoid the requireReinstallation? There is some info about it here. https://stackoverflow.com/questions/40423302/nuget-what-is-the-purpose-of-the-package-requirereinstallation-attribute-in